interop-testing/src/main/proto/io/grpc/testing/integration/messages.proto

```diff
@@ -135,6 +135,10 @@ message StreamingOutputCallRequest {
 message StreamingOutputCallResponse {
   // Payload to increase response size.
   Payload payload = 1;
+
+  // Used to track messages produced as a result of onReady callbacks when
+  // outbound flow-control is opened.
+  int32 on_ready_iteration = 2;
```

Comment:
Making it a separate service might be best. Having this in AbstractTransportTest is interesting as it will raise issues when the transports behave in different ways with regard to isReady/onReady so it would be nice to keep the test X transport behavior we already have.